woof-code/3builddistro

```diff
@@ -493,6 +493,16 @@ for i in $(ls sandbox3/rootfs-complete/usr/share/locale); do
 	case $i in en|en_US|locale.alias) continue ;; esac
 	rm -rf sandbox3/rootfs-complete/usr/share/locale/$i
 done
+
+#overwrite current documentation if DOCS_URL present
+if [ ! -z "$DOCS_URL" ]; then
+	echo "Overwriting current documentation with developer provided $DOCS_URL..."
+	rm -rf sandbox3/rootfs-complete/usr/share/doc
+	wget -t 3 -T 60 -O doc.zip "$DOCS_URL"
```

Comment:
1. Add caching of some sort (download to local-repositories and skip download if already there?)
2. Fail 3builddistro if wget failed

Reply:
@dimkr, please give any example for (1).

Reply:
https://github.com/puppylinux-woof-CE/woof-CE/blob/4c122eac026f65501b4a85c6b8c08a28377ce309/woof-code/support/pc_image.sh#L62

Reply:
@dimkr, I have tried to add a caching system, which saves downloaded files as `../../local-repositories/$DOCS_URL.doc.zip`. But I am unsure if this will work or not, as the link may also contain characters such as `:` and `/`.

Reply:
You can generate a unique file name instead, i.e.`$(echo "$DOCS_URL" | sha256sum | cut -f 1 -d ' ').zip`

Reply:
Or, just take the base name and assume it's unique, i.e. `../../local-repositories/${DOCS_URL##*/}.doc.zip`

Reply:
Updated with a newer caching system.